kodilla-patterns2/src/main/java/com/kodilla/patterns2/observer/homework/Student.java

```diff
@@ -0,0 +1,47 @@
+package com.kodilla.patterns2.observer.homework;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Student implements WatchedStudents {
+
+    private final List<Mentor> mentors;
+    private final List<String> homeworks;
+    private final String name;
+
+    public Student(String name) {
+        mentors = new ArrayList<>();
+        homeworks = new ArrayList<>();
```

Comment:
inicjalizację tych dwóch możesz przenieść odpowiednio do linijek 8 i 9, nie ma potrzeby żeby to było w konstruktorze